Enterprise-CMCS/macpro-mdct-qmr
# [QMR] Tech Spec Changes - AIF-HH

### Description
Text changes for AIF-HH (specifically FFY 2023 measures)

### Related ticket(s)
MDCT-2381

---
### How to test
- Login using the stateuser4@test.com user
- Add health home core data set
- Open health home core data set, navigate to AIF-HH
- See corresponding text changes
    - Title change: Admission to *a Facility* from the community 
    - Measure description change: Choose CMS under "Measurement Specification", navigate down to Performance Measure section and confirm text changes

### Important updates
N/A

---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
